net-libs/pacparser/pacparser-1.3.7-r1.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Why not EAPI=7?

Reply:
Did not try, wanted atomic patches initially and later squashed my 3 into one, to not bump over 3 revisions. Changing the API is something the maintainer can do any time, i am not sure contributors should be asked for that.